### ISSUE TITLE
Allow use of DVID volumes as back-end

### ISSUE THREAD

**Issue by jni** [janelia-flyem/gala#58]
PyDVID might be useful:

- https://github.com/janelia-flyem/pydvid
- https://janelia-flyem.github.io/pydvid/quickstart.html

**Comment by stephenplaza** [comment]
Please do not use pydvid.  It is deprecated.  Use libdvid-cpp which has a libdvid python implementation.  It is fast and up-to-date.

**Comment by jni** [comment]
Noted! Maybe put a big fat "DEPRECATED" sign on the PyDVID docs and readme... =)

**Comment by stephenplaza** [comment]
Fair enough :)  I just added a quick line to libdvid-cpp noting that there are copious examples of the API in the unit tests for both C++ and python API.  At some point, I need to document the python API as well as the C++ one is documented.

In general, the C++ library tries to implement the "best of" DVID API plus some special functions.  We expose much of this through the python API. If you need other things exposed, ask Stuart or me and it should be doable.  Right now there are some heavier-duty stuff not exposed like a parallel tile and sparse block fetcher.
